schema/Readme.md

```diff
@@ -431,6 +431,8 @@ of the model it is pointing to. If you omitted `Writer.blogs`, Prisma would add
 
 You may also map to composite primary keys:
 
+> ⚠ This is not implemented yet.
```

Comment:
Can we put this under the section headline please?

Reply:
I wanted to express that only the composite primary keys are not implemented yet. One to many relationships are implemented. Or should i put this marker in parentheses into the preceding sentence?

Reply:
Should we maybe add a headline before the "You may also map to composite primary keys:" sentence? Then it would make sense again and as this is a slightly different thing, being able to link to its headline makes sense in the first place.

On the other hand: How is that paragraph different than https://github.com/prisma/specs/tree/master/schema#referencing-primary-composite-keys?  If it is not, we should just get rid of _this_ occurence and move the "Underneath ..." stuff from up here to the other one.

Reply:
Followed your second suggestion.